### ISSUE TITLE
Associations icon is Person, not the Association tab's "Hamburger"

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/CardView#231]
The icon used in the Person panels for Associations is a Person icon.  

![image](https://user-images.githubusercontent.com/69127217/204038861-fdcca88f-acce-4181-97ab-b1b4960caba3.png)

The indicator in the Person Editor for the Associations is a "Hamburger"
![image](https://user-images.githubusercontent.com/69127217/204039024-50423025-a4d1-4157-bd82-75880f081db1.png)

**Comment by emyoulation** [comment]
here's the path of one size of the hamburger icon in the Windoze C:\Programs\ folder
GrampsAIO64-5.1.5\share\icons\Adwaita\24x24\actions\open-menu-symbolic.symbolic.png

**Comment by cdhorn** [comment]
Yes, this is inconsistent, but a hamburger icon is absolutely the wrong icon to be using for Associations.  That is what actually needs fixing.

**Comment by emyoulation** [comment]
How about an icon like this: 

https://www.clipartmax.com/middle/m2i8b1d3N4b1m2Z5_inform-association-icon-png/

![image](https://user-images.githubusercontent.com/69127217/224520029-7376f5f5-ddfe-4220-81fd-78ec2a5e6b5c.png)

**Comment by cdhorn** [comment]
If we are able to use that in Gramps core perhaps...  @Nick-Hall any thoughts?  The hamburger icon we're currently using for associations in Gramps core is kind of out of place normally representing an overflow menu or whatever.
